dist/images/ovnkube.sh

```diff
@@ -696,6 +698,56 @@ sb-ovsdb() {
   echo "=============== run sb_ovsdb ========== terminated"
 }
 
+# v3 - Runs ovn-dbchecker on ovnkube-db pod.
+ovn-dbchecker() {
+  trap 'kill $(jobs -p); exit 0' TERM
+  check_ovn_daemonset_version "3"
+  rm -f ${OVN_RUNDIR}/ovn-dbchecker.pid
+  
+  # wait for ready_to_start_node
+  echo "=============== ovn-dbchecker - (wait for ready_to_start_node)"
+  wait_for_event ready_to_start_node
+  echo "ovn_nbdb ${ovn_nbdb}   ovn_sbdb ${ovn_sbdb}"
+
+  # wait for nb-ovsdb and sb-ovsdb to start
+  echo "=============== ovn-dbchecker (wait for nb-ovsdb) ========== OVNKUBE_DB"
+  wait_for_event attempts=15 process_ready ovnnb_db
+
+  echo "=============== ovn-dbchecker (wait for sb-ovsdb) ========== OVNKUBE_DB"
+  wait_for_event attempts=15 process_ready ovnsb_db
+
+  local ovn_db_ssl_opts=""
+  [[ "yes" == ${OVN_SSL_ENABLE} ]] && {
+    ovn_db_ssl_opts="
+        --nb-client-privkey ${ovn_controller_pk}
+        --nb-client-cert ${ovn_controller_cert}
+        --nb-client-cacert ${ovn_ca_cert}
+        --nb-cert-common-name ${ovn_controller_cname}
+        --sb-client-privkey ${ovn_controller_pk}
+        --sb-client-cert ${ovn_controller_cert}
+        --sb-client-cacert ${ovn_ca_cert}
+        --sb-cert-common-name ${ovn_controller_cname}
+      "
+  }
+
+  echo "=============== ovn-dbchecker ========== OVNKUBE_DB"
+  /usr/bin/ovndbchecker \
+    ${K8S_NODE} \
+    --nb-address=${ovn_nbdb} --sb-address=${ovn_sbdb} \
+    --loglevel=${ovnkube_loglevel} \
+    --logfile-maxsize=${ovnkube_logfile_maxsize} \
+    --logfile-maxbackups=${ovnkube_logfile_maxbackups} \
+    --logfile-maxage=${ovnkube_logfile_maxage} \
+    --pidfile ${OVN_RUNDIR}/ovn-dbchecker.pid \
+    --logfile /var/log/ovn-kubernetes/ovn-dbchecker.log \
+    ${ovn_db_ssl_opts} &
```

Comment:
I think ovn_db_ssl_opts is not used either.

Reply:
ovn_db_ssl_opts is needed because the ovn-dbchecker needs access to the config to get to the known members in the raft cluster.

Reply:
from what I can see, only ovn-appctl and ovsdb-tool command is used, neigther of them need ssl options?

Reply:
Check for `config.OvnNorth.Address` is done here https://github.com/ovn-org/ovn-kubernetes/pull/1717/files#diff-e25edc8291296f9c4941674dad166da5R128. So we need the ssl options to be passed in so the config can be read correctly.

Reply:
config.OvnNorth.Address is set with the option -nb-address=${ovn_nbdb}, not the ssl options.

Reply:
good point. I was in my mind clubbing the two together.